package.json

```diff
@@ -25,13 +25,15 @@
     "jsonwebtoken": "^8.3.0",
     "method-override": "^2.3.10",
     "methods": "^1.1.2",
```

Comment:
Please include scripts to run `sequelize migrations for developement, test and production` and also a script to undo migrations.

Reply:
@rovilay Okay, I've fixed it